src/engines/html5/html5.js

```diff
@@ -181,16 +182,57 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   attach(): void {
     Object.keys(Html5Events).forEach((html5Event) => {
       this._eventManager.listen(this._el, Html5Events[html5Event], () => {
-        this.dispatchEvent(new FakeEvent(Html5Events[html5Event]));
+        if (Html5Events[html5Event] === Html5Events.ERROR) {
+          this._handleHTML5Error();
+        } else {
+          this.dispatchEvent(new FakeEvent(Html5Events[html5Event]));
+        }
       });
     });
+
     if (this._mediaSourceAdapter) {
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.VIDEO_TRACK_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.AUDIO_TRACK_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_TRACK_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ABR_MODE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_CUE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
+      this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ERROR, (event: FakeEvent) => this.dispatchEvent(event));
+    }
+  }
+
+  /**
+   * Handles errors from the video element
+   * @returns {void}
+   * @private
+   */
+  _handleHTML5Error(): void {
```

Comment:
No reason to say `_handleHTML5Error`, use `_handleError `

Reply:
handleVideoError?

it's the video tag errors.